src/index.rst

```diff
@@ -612,10 +643,67 @@ Change *LSST_VERSION* in the example yaml accordingly: ::
      inCollection: "HSC/RC2/defaults"
      dataQuery: "exposure = 34342 AND detector = 10"
 
+For different ``butlerConfig`` directory, you also need to grant group permission for PanDA to access the butler::
```

Comment:
I'm guessing this duplication is not important because soon the SDF instructions are going to be deleted.